test/test-files/auth/ldap-smtp-auth-spoofed-sender-with-filter-exception.txt

```diff
@@ -1,6 +1,6 @@
 EHLO mail
 AUTH LOGIN
-c29tZS51c2VyLmVtYWlsQGxvY2FsaG9zdC5sb2NhbGRvbWFpbgo=
+c29tZS51c2VyQGxvY2FsaG9zdC5sb2NhbGRvbWFpbg==
```

Comment:
Why have you made a change here?

Reply:
Indeed, this struck me as well.

Reply:
It is just draft at the moment. So I don't propose these changes. I have yet to figure it out

For me it looks like mistake in master branch. It throws "authentication failure" error. Test passes because it waits "End data with" message. 

It looks weird for me. But maybe I didn't get logic here

Please check the logs:

<details>

```bash
## master unmodified branch
## AUTH LOGIN
## c29tZS51c2VyLmVtYWlsQGxvY2FsaG9zdC5sb2NhbGRvbWFpbgo=
## c2VjcmV0

root@mail:/# nc 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/ldap-smtp-auth-spoofed-sender-with-filter-exception.txt 
220 mail.my-domain.com ESMTP
250-mail.my-domain.com
250-PIPELINING
250-SIZE 10240000
250-ETRN
250-STARTTLS
250-AUTH PLAIN LOGIN
250-AUTH=PLAIN LOGIN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250-DSN
250 CHUNKING
334 VXNlcm5hbWU6
334 UGFzc3dvcmQ6
535 5.7.8 Error: authentication failed: authentication failure
250 2.1.0 Ok
250 2.1.5 Ok
354 End data with <CR><LF>.<CR><LF> # <---
250 2.0.0 Ok: queued as 2795C460078
221 2.0.0 Bye

## master branch with modified login
## some.user.email@localhost.localdomain -> some.user@localhost.localdomain
## AUTH LOGIN
## c29tZS51c2VyQGxvY2FsaG9zdC5sb2NhbGRvbWFpbg==
## c2VjcmV0

root@mail:/# nc 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/ldap-smtp-auth-spoofed-sender-with-filter-exception.txt 
220 mail.my-domain.com ESMTP
250-mail.my-domain.com
250-PIPELINING
250-SIZE 10240000
250-ETRN
250-STARTTLS
250-AUTH PLAIN LOGIN
250-AUTH=PLAIN LOGIN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250-DSN
250 CHUNKING
334 VXNlcm5hbWU6
334 UGFzc3dvcmQ6
235 2.7.0 Authentication successful
250 2.1.0 Ok
553 5.7.1 <randomspoofedaddress@localhost.localdomain>: Sender address rejected: not owned by user some.user@localhost.localdomain
554 5.5.1 Error: no valid recipients
221 2.7.0 Error: I can break rules, too. Goodbye.
```

</details>